dedupe/record_linkage_testing/Dockerfile.api

```diff
@@ -0,0 +1,13 @@
+ARG VERSION=v1.2.11
+FROM ghcr.io/cdcgov/phdi/record-linkage:${VERSION}
```

Comment:
I haven't ran this yet but did you need to create a PAT token to use this?

Reply:
I didn't, the record-linkage package repo is public.